src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -805,6 +805,8 @@ private function startClass($class, $baseClass, $namespace)
  *
  * This class has been auto-generated
  * by the Symfony Dependency Injection Component.
+ *
+ * @final since Symfony 3.3
```

Comment:
Should we add a dot at the end to be consistent with [`@deprecated`](https://github.com/symfony/symfony/blob/ea973feeb1687e7119073ec0f45c1e903799e325/src/Symfony/Component/Security/Core/Authorization/DebugAccessDecisionManager.php#L17)?